resources/apa.csl

```diff
@@ -130,6 +130,9 @@
           <if variable="DOI">
             <text variable="DOI" prefix="doi:"/>
           </if>
+          <else-if variable="ISBN">
+            <text variable="ISBN" prefix="ISBN:&#160;"/>
```

Comment:
@tarleb - what's the reason for encoding the space like this? I notice that the `doi:` one above doesn't do this?

Reply:
There is no space at all after *doi:*. My understanding is that it is meant to resemble an URI, and doi a IATA registered scheme. This does not apply to ISBN, so a non-breaking space is used. This will prevent line-breaks between the "ISBN" literal and the actual ISBN number.